.gitignore

```diff
@@ -189,6 +189,45 @@ Makefile.am-stamp
 /src/glue/romio/all_romio_symbols.c
 /src/mpi/coll/src/csel_json_autogen.c
 
+# Files generated by autogen binding scripts
+/src/binding/c/attr/
+/src/binding/c/coll/
+/src/binding/c/comm/
+/src/binding/c/datatype/
+/src/binding/c/errhan/
+/src/binding/c/errnames.txt
+/src/binding/c/group/
+/src/binding/c/info/
+/src/binding/c/init/
+/src/binding/c/misc/
+/src/binding/c/mpit/
+/src/binding/c/op/
+/src/binding/c/request/
+/src/binding/c/rma/
+/src/binding/c/spawn/
+/src/binding/c/timer/
+/src/binding/c/topo/
+/src/include/mpi_proto.h
+/src/include/mpir_impl.h
+/src/mpi_t/qmpi_register.h
+/src/mpid/ch4/netmod/include/netmod.h
+/src/mpid/ch4/netmod/include/netmod_impl.h
+/src/mpid/ch4/netmod/ofi/func_table.c
+/src/mpid/ch4/netmod/ofi/ofi_noinline.h
+/src/mpid/ch4/netmod/src/netmod_impl.c
+/src/mpid/ch4/netmod/stubnm/func_table.c
+/src/mpid/ch4/netmod/stubnm/netmod_inline.h
+/src/mpid/ch4/netmod/stubnm/stubnm_noinline.c
+/src/mpid/ch4/netmod/stubnm/stubnm_noinline.h
+/src/mpid/ch4/netmod/ucx/func_table.c
+/src/mpid/ch4/netmod/ucx/ucx_noinline.h
+/src/mpid/ch4/shm/include/
+/src/mpid/ch4/shm/posix/posix_noinline.h
+/src/mpid/ch4/shm/src/shm_noinline.h
+/src/mpid/ch4/shm/stubshm/shm_inline.h
+/src/mpid/ch4/shm/stubshm/shm_noinline.h
+/src/mpid/ch4/shm/stubshm/stubshm_noinline.c
+
```

Comment:
The `.gitignore` is already updated in `main`, you can remove these changes now -- just `/src/mpi_t/qmpi_register.h`.